pygmt/datasets/load_remote_dataset.py

```diff
@@ -147,6 +147,26 @@ class GMTRemoteDataset(NamedTuple):
             "02m": Resolution(["pixel"], True),
         },
     ),
+    "earth_vgg": GMTRemoteDataset(
+        title="Earth vertical gravity gradient",
+        name="earth_vgg",
+        long_name="IGPP Global Earth Vertical Gravity Gradient",
+        units="Eotvos",
+        extra_attributes={"horizontal_datum": "WGS84"},
+        resolutions={
+            "01d": Resolution(["pixel", "gridline"], False),
+            "30m": Resolution(["pixel", "gridline"], False),
+            "20m": Resolution(["pixel", "gridline"], False),
+            "15m": Resolution(["pixel", "gridline"], False),
+            "10m": Resolution(["pixel", "gridline"], False),
+            "06m": Resolution(["pixel", "gridline"], False),
+            "05m": Resolution(["pixel", "gridline"], True),
+            "04m": Resolution(["pixel", "gridline"], True),
+            "03m": Resolution(["pixel", "gridline"], True),
+            "02m": Resolution(["pixel", "gridline"], True),
+            "01m": Resolution(["gridline"], True),
```

Comment:
See https://www.generic-mapping-tools.org/remote-datasets/earth-vgg.html
```suggestion
            "01m": Resolution(["pixel"], True),
```